src/bundles/ApplicationsAdmin/components/Agency/Agency.js

```diff
@@ -43,6 +43,11 @@ class Agency extends React.Component {
     } else {
       data.whitelisted = false
     }
+    if (data.must_join_team === "on") {
```

Comment:
Where is this `on` value coming from?

Reply:
This is the value coming back from the checkbox component (on or off) - it's consistent with the other radios in the same form in the lines above this.